app/notify_client/service_api_client.py

```diff
@@ -532,5 +535,12 @@ def get_notification_count(self, service_id):
 
         return int(count)
 
+    def get_global_notification_count(self):
+        # if cache is not set, or not enabled, return 0
+
```

Comment:
Small styling nit-pick:  no need for the extra line here.

```suggestion
```

Reply:
I removed the empty white space as well as the one in the function above.